### PR TITLE
Implemented solr query to fetch and show data in orderDetails page( #1x634dq )

### PR DESCRIPTION
**Pull request by meet-aniket**

### CODE REVIEW

File: src/views/Orders.vue [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+        "json": {
+        "params": {
+          "rows": viewSize,
+          "group": true,
+          "group.field": "orderId",
+          "group.limit": 10000
+        },
+        "query": "docType:OISGIR",
+        "fields": ""
+      }
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:253]: Improve indentation.

File: src/store/modules/order/actions.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  // Find Orders
+  async findOrders ( { commit, state }, payload) {
+    let resp;
+    try {
+      resp = await OrderService.findOrder(payload)
+      if (resp && resp.status === 200 && !hasError(resp)) {
+        const orders = resp.data.grouped.orderId;
+        this.dispatch('product/getProductInformation', { orders });
+        commit(types.ORDER_LIST_UPDATED, {
+          items: orders.groups,
```
Comment (bashu22tiwari) [inline src/store/modules/order/actions.ts:20]: Improve indentation.

File: src/views/Orders.vue [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+  IonList,
+  IonListHeader,
+  IonNote,
+  IonPage,
+  IonSearchbar,
+  IonSelect,
+  IonThumbnail,
+  IonTitle,
+  IonToolbar,
+  IonSelectOption,
```
Comment (bashu22tiwari) [inline src/views/Orders.vue:194]: Order them alphabetically.

**Comment by bashu22tiwari** [comment]
@meet-aniket Just check once if you have internationalized all the static text.

**Comment by bashu22tiwari** [comment]
@meet-aniket , also I guess the orders on the right-hand side are overlapping according to the below Screenshots. Please confirm once, and resolve accordingly.
Screenshot from 2022-01-25 01-31-38

**Comment by meet-aniket** [comment]
This pull request was created when the UI of FindOrders.vue page was not complete and multiple commits are there in the pull request which does not make any sence.

**Pull request closed without merging**